dynamic_rest/serializers.py

```diff
@@ -301,6 +317,29 @@ def get_name(cls):
         return cls.Meta.name
 
     @classmethod
+    def get_url(self, pk=None):
+        # if associated with a registered viewset, use its URL
+        url = getattr(self, '_url', None)
+        if url:
+            # use URL key to get endpoint
+            url = reverse(url)
+        if not url:
+            # otherwise, return canonical URL for this model
+            from dynamic_rest.routers import DynamicRouter
+            url = DynamicRouter.get_canonical_path(
+                self.get_resource_key()
+            )
+        if pk:
+            return '%s/%s/' % (url, pk)
+        return url
+
+    @classmethod
+    def get_natural_key(cls):
+        if not hasattr(cls.Meta, 'natural_key'):
+            return 'name'
```

Comment:
TODO(ant): more reasonable fallback behavior, e.g. look for a unique field on the model, or look through model fields for something unique